src/main/java/org/fundacionjala/coding/Main.java

```diff
@@ -0,0 +1,14 @@
+package org.fundacionjala.coding;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
Since now you will use unit tests, this class will not be longer necessary.
Please remove it!.